File: backend/entity/Condition.go
package entity

import (
	"gorm.io/gorm"
	"time"
)

type Condition struct {
	gorm.Model
	Day    time.Time   
	Price  float32
	Accommodation string
	Landmark string
	User_id  uint      
   	User    *User  `gorm:"foreignKey:User_id"`
}

File: frontend/src/interfaces/Condition.ts
export interface ConditionInterface {
    ID?: number;
    Day?: string; // Use string to represent date in 'YYYY-MM-DD' format // Use string to represent time in 'HH:mm:ss' format
    Price?: number; // Use number for price
    Accommodation?: string; // Accommodation name or ID
    Landmark?: string; // Landmark name or ID
    Style?: string; // Style of the condition (e.g., budget, luxury)
    User_id?: number; // User ID who created the condition
}
// Day    time.Time   
// 	Price  float32
// 	Accommodation string
// 	Landmark string
// 	User_id  uint      